tests/grant.test.ts

```diff
@@ -27,10 +28,36 @@ export function runTests(): void {
 		assert.assertNotNull(WorkspaceMember.load(mem!.member!))
 	})
 
+	test('should create a grant with custom token reward', () => {
```

Comment:
the token assertions can just be moved into the create grant test -- since this is test is just a copy of the aforementioned test

Reply:
Okay